src/explore-education-statistics-frontend/src/modules/table-tool/TableToolPage.tsx

```diff
@@ -447,6 +449,47 @@ class TableToolPage extends Component<Props, State> {
 
                       {publication && createdTable && (
                         <>
+                          <h3>Share your table</h3>
+                          <ul className="govuk-list">
+                            <li>
+                              {permalinkId ? (
+                                <>
+                                  <div>Generated permanent link:</div>
+                                  <LinkContainer
+                                    url={`${window.location.href}/permalink/${permalinkId}`}
+                                  />
+                                  <div>
+                                    <a
+                                      className="govuk-link"
+                                      href={`${window.location.href}/permalink/${permalinkId}`}
+                                      title="View created table permalink"
+                                      target="_blank"
+                                      rel="noopener noreferrer"
+                                    >
+                                      View permanent link
+                                    </a>
```

Comment:
I'm thinking we should still show more than just a view link here. Probably a text box with the link so you can copy it but i don't think that holds up this PR and we can raise a separate enhancement to cover it :)